src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+//  looking for span element value by class name
+const elements = document.getElementsByClassName('population');
+//  making array that filed by element value
+const data = [].map.call(elements, elem => elem.textContent);
+//  deleting comma in array items
+const newArr = data.map(stringItem => stringItem.replace(/[,]/g, ''));
```

Comment:
create function parseNumber
```suggestion
const newArr = data.map(parseNumber);
```